### ISSUE TITLE
pass Travis CI tests

### ISSUE THREAD

**Issue by twhiteaker** [twhiteaker/pynwm#5]
local tests run fine, but Travis miniconda fails in netCDF4.date2num.

**Comment by twhiteaker** [comment]
```
src/pynwm/nwm_subset.py:133: in combine_files

    time_var[:] = [round(d) for d in date2num(t, time_units)]

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???

E   TypeError: can't subtract offset-naive and offset-aware datetimes
```

Each date in variable `t` is offset aware. I suspect the time_units is being used to create a datetime by date2num. I've tried units of `minutes since 1970-01-01 00:00:00-00` and `minutes since 1970-01-01 00:00:00 UTC`, but neither solved the problem.

**Comment by twhiteaker** [comment]
In netCDF4 version 1.1.6, they changed the units parser so it returns time-zone naive datetimes. Therefore, you must provide time-zone naive datetimes to date2num.

**Comment by twhiteaker** [comment]
Fixed in 57334f52566497aeceba56237a2a19aab1a4d1ea
